boards/arm/pan1780_evb/pan1780_evb_defconfig

```diff
@@ -0,0 +1,44 @@
+# PAN1780 EVB configuration
+
+# Copyright (c) 2022 Panasonic Industrial Devices Europe GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_PAN1780_EVB=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
+
+# Please activate NFC following NFC config on pan1780_evb
+#CONFIG_NFC_T2T_NRFXLIB=y
+#CONFIG_NFC_NDEF=y
+#CONFIG_NFC_NDEF_MSG=y
+#CONFIG_NFC_NDEF_RECORD=y
+#CONFIG_NFC_NDEF_URI_MSG=y
+#CONFIG_NFC_NDEF_URI_REC=y
+
+# bluetooth advertising name
+CONFIG_BT_DEVICE_NAME="pan1780_evb"
```

Comment:
boards only enable minimal peripherals, these options seem to belong to the application domain

Reply:
Fixed